rest-api-model-server/src/test/kotlin/ReplicatedRepositoryInstantiationTest.kt

```diff
@@ -0,0 +1,30 @@
+package org.acme.getting.started.testing
+
+import io.quarkus.test.junit.QuarkusTest
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Assertions.assertThrows
+import org.junit.jupiter.api.Test
+import org.modelix.model.client.ReplicatedRepository
+import javax.inject.Inject
+
+@QuarkusTest
+class ReplicatedRepositoryInstantiationTest {
+    @Inject
+    lateinit var replicatedRepository: ReplicatedRepository
+
+    @Test
+    fun testInjection() {
+        // This tests, whether `replicatedRepository` can be correctly injected and instantiated by Quarkus.
+        // The actual instantiation is happening when we call something on `replicatedRepository`.
+        // Therefore, we call `.branch`.
+        // The asserted exception message indicates that the `replicatedRepository` was injected correctly.
+        // The exception message "java.lang.NoClassDefFoundError: kotlin/enums/EnumEntriesKt", for example,
+        // would indicate that something is set up wrong and the instantiation is failing.
+        // In the past, we had such issues with the wrong version of Kotlin and Ktor.
+        assertThrows(
+            RuntimeException::class.java,
+            { replicatedRepository?.branch },
```

Comment:
```suggestion
            { replicatedRepository.branch },
```

It's not nullable anymore with lateinit

Reply:
Done ^^'

I relied too long on linting on commits